pkg/providers/v1/providers.go

```diff
@@ -56,6 +56,8 @@ type Provider interface {
 	// The name is used to identify the entity within minder and is how
 	// it will be stored in the database.
 	GetEntityName(entType minderv1.Entity, props *properties.Properties) (string, error)
+	// OperationalProperties returns the operational properties for an entity
+	OperationalProperties(entType minderv1.Entity) []string
```

Comment:
Question: Can we come up with a way to expose the operational properties without adding a new method to the generic Provider interface? It feels like the interface is getting too big already.

Reply:
Why do we need to expose or filter them?

Reply:
We need to filter them because at the same time we need to be able to remove properties that were deleted upstream but we need to retain the operational attributes like hook ID.
Anyway, we discussed that offline and agreed to move the attributes handling to the provider completely so that minder-core is oblivious to their existence.